tutorials/clinical/10_ieeg_localize.py

```diff
@@ -0,0 +1,373 @@
+Contact locations in MR-space are the goal because this is the image from which
+brain structures can be determined using the freesurfer reconstruction
+:ref:`tut-freesurfer-reconstruction`. Contact locations in MR-space can also
+be translated to an average space such as ``fsaverage`` for group comparisons.
+"""
+
+# Authors: Alex Rockhill <aprockhill@mailbox.org>
+#          Eric Larson <larson.eric.d@gmail.com>
+#
+# License: BSD (3-clause)
+
+import os.path as op
+
+import numpy as np
+import pandas as pd
+import matplotlib.pyplot as plt
+
+import nibabel as nib
+from nibabel.processing import resample_from_to
+from nipy.algorithms.registration.histogram_registration import (
+    HistogramRegistration)
+import mne
+from mne.datasets import fetch_fsaverage
+
+print(__doc__)
+
+np.set_printoptions(suppress=True)  # suppress scientific notation
+
+# paths to mne datasets - sample sEEG and FreeSurfer's fsaverage subject
+# which is in MNI space
+misc_path = mne.datasets.misc.data_path()
+sample_path = mne.datasets.sample.data_path()
+subject = 'fsaverage'
+subjects_dir = op.join(sample_path, 'subjects')
+
+# use mne-python's fsaverage data
+fetch_fsaverage(subjects_dir=subjects_dir, verbose=True)  # downloads if needed
+
+###############################################################################
+# Aligning the T1 to ACPC
+# =======================
+#
+# Let's load our T1-weighted image and visualize it. As you can see, the
+# image is already aligned to the anterior commissue and posterior commissure
+# (ACPC). This is recommended to do before starting. This can be done using
+# freesurfer's freeview:
+#
+# .. code-block:: bash
+#
+#     $ freeview $MISC_PATH/seeg/sample_seeg_T1.mgz
+#
+# And then interact with the graphical user interface:
+#
+# First, it is recommended to change the cursor style to long, this can be done
+# through the menu options like so:
+#
+#     ``Freeview -> Preferences -> General -> Cursor style -> Long``
+#
+# Then, the image needs to be aligned to ACPC to look like the image below.
+# This can be done by pulling up the transform popup from the menu like so:
+#
+#     ``Tools -> Transform Volume``
+#
+# .. note::
+#     Be sure to set the text entry box labeled RAS (not TkReg RAS) to
+#     ``0 0 0`` before beginning the transform.
+#
+# Then translate the image until the crosshairs meet on the AC and
+# run through the PC as shown in the plot. The eyes should be in
+# the ACPC plane and the image should be rotated until they are symmetrical,
+# and the crosshairs should transect the midline of the brain.
+# Be sure to use both the rotate and the translate menus and save the volume
+# after you're finished using ``Save Volume As`` in the transform popup
+# :footcite:`HamiltonEtAl2017`.
+
+T1 = nib.freesurfer.load(op.join(misc_path, 'seeg', 'sample_seeg_T1.mgz'))
+viewer = T1.orthoview()
+viewer.set_position(0, 9.9, 5.8)
+viewer._axes[0].annotate('PC', (107, 108), xytext=(10, 75),
```

Comment:
Do not use private attributes in examples, can this be accessed via `viewer.figure.axes[0]` for example? Or maybe `viewer.fig.axes[0]`?